test/Microsoft.Extensions.FileProviders.Physical.Tests/ExcludedFilesTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.Extensions.FileProviders.Physical.Internal;
+using Xunit;
+
+namespace Microsoft.Extensions.FileProviders.Physical.Tests
+{
+    public class ExcludedFilesTests : IDisposable
+    {
+        private readonly DisposableFileSystem _fileSystem = new DisposableFileSystem();
+
+        public void Dispose()
+        {
+            _fileSystem.Dispose();
+        }
+
+        [Theory]
+        [MemberData(nameof(Combinations))]
+        public void FiltersExcludedFiles(string filename, FileAttributes attributes, ExcludedFiles excludedFiles, bool excluded)
```

Comment:
Not sure I understand the request. Am I missing scenarios? Or is this just a coding preference?